container/openejb-jee/src/test/java/org/apache/openejb/jee/JeeTest.java

```diff
@@ -18,23 +18,22 @@
 
 package org.apache.openejb.jee;
 
+import jakarta.xml.bind.*;
```

Comment:
Can we restore the original import order? It is basically a style change

Reply:
I restored the original order, but I have not commited yet.